### PR TITLE
added images for intro 3 and 4

### PR DESCRIPTION
**Pull request by arunraja-hub**

### CODE REVIEW

**Comment by adpunt** [comment]
Looks great! Although I don't know if we need the image description "The DMTA Cycle", it seems self-explanatory

**Comment by arunraja-hub** [comment]
Yeah i added since we had captions for all images on the intro pages

File: src/components/body/navigation.css [hunk 1]
```diff
@@ -55,7 +55,7 @@
 }
 
 .navigation-link.active {
-    color: #9a9c9e;
+    color: #30D5C8;
```
Comment (adpunt) [inline src/components/body/navigation.css:58]: Potentially try BBD9E7, it may be a bit more subtle but still distinguishes enough from the grey

**Comment by adpunt** [comment]
> Yeah i added since we had captions for all images on the intro pages

I think for the others it makes sense since its a little less clear that its a protein structure of the target compound, but at least for this image there's text

File: src/components/body/navigation.css (hunk 1 above)
Comment (anissa218) [reply src/components/body/navigation.css:58]: Yeah we know garrett said so but i don't think the bright turquoise looks that good esthetically

**Review by SimiaoZhao — APPROVED** [review]

**Pull request merged**